appengine/search/google-checks.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0"?>
```

Comment:
We shouldn't need this. If you use the same parent as the other samples, it should use the config in the java-repo-tools directory.